jvm-packages/xgboost4j-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/Utils.scala

```diff
@@ -16,15 +16,6 @@
 
 package ml.dmlc.xgboost4j.scala.spark
 
-import java.io.File
-
 trait Utils extends Serializable {
```

Comment:
It is related by the fact that external cache reuses SparkContext data :)

Anyway, I've pushed the change.